src/components/Layout/PhoneNumberInput.tsx

```diff
@@ -63,6 +76,7 @@ export const PhoneNumberInput: FunctionComponent<{
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   onSubmit = () => {}
 }) => {
+  const value: string = phoneNumberFormatted(mobileNumberValue);
```

Comment:
Have you considered using `useState` to maintain this value since it is being altered in this component?

This approach works now because `onChangeMobileNumber` in the parent component actually updates the value and passes it back down, which reinstaniates this `const` with the new value.

Instantiating this as `const` also makes it such that this value is immutable, which is not the case in this component.

Using `useState` will allow us to easily manipulate this value in the component, as well as leverage React's state management if need be.

Reply:
but if `useState` then the regex in loginMobileNumberCard will fail. Have initially considered using `useState` to modify the formatted value in the mobileNumberValue.

but this approach will require cleaning up the data when submitting the data in the loginMobileNumberCard. For example, removing the space, then have to modify in loginMobileNumberCard.